src/cobalt_model/pagination_config.rs

```diff
@@ -4,8 +4,11 @@ use std::vec::Vec;
 use super::SortOrder;
 
 pub const DEFAULT_PERMALINK_SUFFIX: &str = "{{num}}/";
-pub const DEFAULT_SORT: &str = "published_date";
+
 pub const DEFAULT_PER_PAGE: i32 = 10;
+lazy_static! {
```

Comment:
Reminds me that I need to go back sometime and move from lazy_static to once_cell

Reply:
Just discovering `once_cell`, looks really great!